examples/lab-minimal/v2/qwiklabs.yml

```diff
@@ -21,6 +21,10 @@ instruction:
 
 environment:
   student_visible_outputs:
+    - label:
+        locales:
+          en: "Open GCP Console"
+      reference: primary_project.console_url
```

Comment:
While the definition of "minimal" is rather subjective, I'd recommend also including a gcp user username + password here as that's what a "typical" V1 converted lab would look like.

Reply:
It does already, doesn't it?  Lines 32-39?

Reply:
I could have sworn I looked down there but you're absolutely right.